style.css

```diff
@@ -1,69 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
-  margin: auto;
+  font-family: "Open Sans", sans-serif;
+  padding-bottom: 50px;
 }
 
-.nav-bar .nav-container {
-  background-color: #000000;
-  color: #ffffff;
+.page-bg {
+  background-image: url("https://images.pexels.com/photos/450035/pexels-photo-450035.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+  height: 100vh;
+  width: 100vw;
```

Comment:
This is not needed. This makes the screen look odd. You can see that there is gap in opacity and the actual image. 

<img width="964" alt="Screen Shot 2022-11-22 at 8 26 17 AM" src="https://user-images.githubusercontent.com/12387239/203325429-9b8b4963-d02d-4144-9a7a-b17888a54eaf.png">

Reply:
fixed.